updates/helper/updates.go

```diff
@@ -20,6 +27,26 @@ func PlatformIdentifier(identifier string) string {
 // MandatoryUpdates returns mandatory updates that should be loaded on install
 // or reset.
 func MandatoryUpdates() (identifiers []string) {
+	// Intel
+	identifiers = append(
+		identifiers,
+
+		// Filter lists data
+		"all/intel/lists/index.dsd",
+		"all/intel/lists/base.dsdl",
+		"all/intel/lists/intermediate.dsdl",
+		"all/intel/lists/urgent.dsdl",
+
+		// Geo IP data
+		"all/intel/geoip/geoipv4.mmdb.gz",
+		"all/intel/geoip/geoipv6.mmdb.gz",
+	)
+
+	// Stop here if we only want intel data.
+	if intelOnly.IsSet() {
+		return
+	}
+
 	// Binaries
 	if onWindows {
 		identifiers = []string{
```

Comment:
That must be an `append()` now! Otherwise if `!intelOnliy.IsSet()` than intel data is not part of the identifiers anymore!